server/src/main/java/com/objectcomputing/checkins/services/teammembers/TeamMember.java

```diff
@@ -0,0 +1,78 @@
+package com.objectcomputing.checkins.services.teammembers;
+
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+
+@Entity
+@Table(name ="team_member")
+public class TeamMember {
+
+    public TeamMember(UUID teamId, UUID memberId, boolean isLead) {
+                        this.teamId=teamId;
+                        this.memberId=memberId;
+                        this.isLead=isLead;
+                        }
```

Comment:
Another attack of the heinous indentation...just indent these like a normal function.

Reply:
Updated indentation